src/ngx_http_small_light_gd.c

```diff
@@ -146,17 +146,11 @@ ngx_int_t ngx_http_small_light_gd_process(ngx_http_request_t *r, ngx_http_small_
         return NGX_ERROR;
     }
 
+    /* adjust image size */
     iw = gdImageSX(src);
     ih = gdImageSY(src);
     ngx_http_small_light_calc_image_size(r, ctx, &sz, iw, ih);
 
-    /* pass through. */
```

Comment:
Because `pt_flg` is not correct at this point.
Image with `angle` is unreckoned in `ngx_http_small_light_calc_image_size()`.
So we need the actual rotation before the correct `sz`.
(I tried to revise `ngx_http_small_light_calc_image_size()`,  but too hard for me)